actions/views/login.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as UserActions from 'mattermost-redux/actions/users';
```

Comment:
if we do `import *` webpack would include whole file for the login route . It would be better to import specific actions instead.

Reply:
It actually still loads the whole file regardless of how you import it since it needs to run through the rest of the file to define the exports and such. The reason I imported it all instead of specific actions is just to avoid naming conflicts since those will come up a lot when we're implementing actions that just wrap the mattermost-redux versions